core/snippets/snippets/constructorDeclaration.snippet

```diff
@@ -12,7 +12,7 @@ constructor($1) {
 
 language: java
 -
-$modifiers $1($2) {
```

Comment:
Did we intentionally lose a parameter here? The spot for modifiers appears to be gone

Reply:
Yes. Having two snippet holes only separated by space isn't really that practical. When I use these snippets I substituted the modifiers before inserting the snippet.